tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/websub/WebSubContentTypeSupportTestCase.java

```diff
@@ -1,69 +1,87 @@
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
 package org.ballerinalang.test.service.websub;
 
-import io.netty.handler.codec.http.HttpHeaderNames;
-import org.awaitility.Duration;
-import org.ballerinalang.test.BaseTest;
-import org.ballerinalang.test.context.BMainInstance;
 import org.ballerinalang.test.context.BServerInstance;
 import org.ballerinalang.test.context.BallerinaTestException;
 import org.ballerinalang.test.context.LogLeecher;
-import org.ballerinalang.test.util.HttpClientRequest;
-import org.ballerinalang.test.util.HttpResponse;
-import org.ballerinalang.test.util.HttpsClientRequest;
-import org.ballerinalang.test.util.TestConstant;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import java.io.File;
-import java.net.ConnectException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.Executors;
 
-import static org.awaitility.Awaitility.given;
-import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.CONTENT_TYPE_JSON;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.CONTENT_TYPE_STRING;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.CONTENT_TYPE_XML;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.HUB_MODE_INTERNAL;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.HUB_MODE_REMOTE;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.PUBLISHER_NOTIFY_URL;
+import static org.ballerinalang.test.service.websub.WebSubTestUtils.requestUpdate;
 
 /**
  * This class tests support for WebSub notifications of different content types.
  * Tests:
  * 1. Content delivery to multiple subscribers
  * 2. Content delivery for subscriptions made with and without specifying a secret
- * 3. Content delivery for string and XML types. Other WebSub tests cover JSON content.
+ * 3. Content delivery for JSON, string and XML types.
  */
-public class WebSubContentTypeSupportTestCase extends BaseTest {
+@Test(groups = "websub-test")
+public class WebSubContentTypeSupportTestCase extends WebSubBaseTest {
     private BServerInstance webSubSubscriber;
-    private BMainInstance webSubPublisher;
-
-    private final int servicePort = 8686;
 
-    private static String hubUrl = "https://localhost:9696/websub/hub";
     private static final String INTENT_VERIFICATION_SUBSCRIBER_ONE_LOG = "ballerina: Intent Verification agreed - Mode"
-            + " [subscribe], Topic [http://www.websubpubtopic.com], Lease Seconds [3000]";
+            + " [subscribe], Topic [http://one.websub.topic.com], Lease Seconds [3000]";
```

Comment:
The convention which we use is to break the line after the operator.

Reply:
Changed.

Reply:
Seems like the change was not pushed?

Reply:
Isn't https://github.com/ballerina-platform/ballerina-lang/pull/10414/commits/8edb2f6ea66c747f2966d6ab2b465a6752245f81#diff-11480b9d6303890d3c23691b1a1c3807R49 the change expected?

Reply:
Yes, these are the expected changes. Maybe the page wasn't refreshed properly.